src/list-grid.scss

```diff
@@ -0,0 +1,301 @@
+  $fd-grid-list-item-hover-background-color: var(--sapList_Hover_Background);
+  $fd-grid-list-item-hover-box-shadow: 0 0 0 0.0625rem var(--fdGrid_List_Item_Hover_Border_Color);
+  $fd-grid-list-highlight: (
+    'positive': ('backgroundColor': var(--sapSuccessBorderColor)),
+    'negative': ('backgroundColor': var(--sapErrorBorderColor)),
+    'critical': ('backgroundColor': var(--sapWarningBorderColor)),
+    'neutral': ('backgroundColor': var(--sapNeutralBorderColor))
+  );
+
+  @include fd-reset();
+  @include fd-flex(column);
+
+  &__item {
+    @include fd-reset();
+    @include fd-flex(column);
+    @include fd-fiori-focus();
+
+    background-color: var(--sapList_Background);
+    border-radius: $fd-grid-list-border-radius;
+    box-shadow: $fd-grid-list-item-box-shadow;
+    height: 100%;
+    position: relative;
+
+    .#{$block}__radio-input {
+      @include fd-focus() {
+        + .#{$block}__radio-label {
+          outline-offset: 0;
+        }
+      }
+
+      + .#{$block}__radio-label {
+        @include fd-empty() {
+          padding: 0;
+        }
+
+        &::after {
+          top: 0;
+          right: 0 !important;
+          bottom: 0;
+          left: 0;
+          margin: auto !important;
+        }
+      }
+    }
+
+    .#{$block}__checkbox-input {
+      @include fd-focus() {
+        + .#{$block}__checkbox-label {
+          outline-offset: 0;
+        }
+      }
+
+      + .#{$block}__checkbox-label {
+        @include fd-empty() {
+          padding: 0;
+        }
+      }
+    }
+
+    .#{$block}__checkbox-label,
+    .#{$block}__radio-label {
+      display: none;
+    }
+
+    &--unread {
+      .#{$item}-body * {
+        font-weight: bold;
+      }
+    }
+
+    &--link {
+      cursor: pointer;
+
+      @include fd-hover() {
+        box-shadow: $fd-grid-list-item-hover-box-shadow;
+        background-color: $fd-grid-list-item-hover-background-color;
+      }
+
+      @include fd-active() {
+        background-color: $fd-grid-list-item-active-background-color !important;
+
+        * {
+          color: $fd-grid-list-active-text-color !important;
+        }
+      }
+
+      @include fd-navigated() {
+        &::after {
+          content: "";
+          position: absolute;
+          top: 0;
+          right: 0;
+          height: 100%;
+          width: 0.1875rem;
+          display: block;
+          border-radius: 0 $fd-grid-list-border-radius $fd-grid-list-border-radius 0;
+          background-color: $fd-grid-list-selection-border-color;
+        }
+      }
+    }
+  }
+
+  &__item-body {
+    @include fd-reset();
+
+    @include fd-flex(column) {
+      justify-content: space-between;
+    }
+
+    height: 100%;
+    padding: 1rem;
+
+    &---no-padding {
+      padding: 0;
+    }
+  }
+
+  &__item-body-actions {
+    margin-top: 1rem;
+    order: 1;
+  }
+
+  &__item-counter {
+    @include fd-fiori-focus(0);
+
+    color: var(--sapContent_MarkerTextColor);
+  }
+
+  .#{$block}__toolbar {
```

Comment:
should be renamed to `__item-toolbar` as it's part of the item

Reply:
fixed